README.rst

```diff
@@ -4,19 +4,17 @@ NuoDB Client Package
 
 .. contents::
 
-This project bundles the latest publicly-available versions of NuoDB_
-database clients into a single downloadable file.  The official versions of
-the NuoDB Client Package file are available both from `GitHub Releases`_ as
-well as from the `NuoDB Client Portal`_.
+The NuoDB Client drivers are used to connect Client applications to a NuoDB database. The NuoDB Client Package bundles the latest publicly-available NuoDB_ database client driver verions into a single downloadable file. The official versions of
```

Comment:
Also I think re-using the word "cient" in "client applications" and in the "client package" is confusing.

If we need to talk about them, can we use the term "user applications" instead?

Reply:
changed client to user.